docs/configuration.rst

```diff
@@ -190,6 +190,21 @@ below:
 
          **Note** that the JSON object must be included in single quotes (') to avoid problems with escaping of the double quote (") character.
 
+   DD_SPAN_SAMPLING_RULES:
```

Comment:
Should we also document: `DD_SPAN_SAMPLING_RULES_FILE`?

Reply:
ah good call!